Makefile

```diff
@@ -16,14 +16,25 @@ ZMQ_VERSION ?= 4.3.1
 ZMQ_BASE_BUILD_DIR = $(CURDIR)/libzmq/build
 ZMQ_BUILD_DIR = $(ZMQ_BASE_BUILD_DIR)/v$(ZMQ_VERSION)
 ZMQ_PKG_CONFIG_DIR = $(ZMQ_BUILD_DIR)/lib/pkgconfig
+ifeq ($(OS),Windows_NT)
+ZMQ_COMPILE_HOST ?= x86_64-w64-mingw32
```

Comment:
Actually I think we should rename this to `ZMQ_BUILD_HOST` since it is more standard and we already have `ZMQ_BUILD_DIR`.